### ISSUE TITLE
grun init fails v0.11.2

### ISSUE THREAD

**Issue by lfender6445** [TryGhost/Ghost#7614]
Having trouble installing as `grunt init` fails with error: 

```
Error: ENOENT: no such file or directory, open '../../.git/modules/core/client/commondir'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at findRepoHandleLinkedWorktree (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli-app-version/node_modules/git-repo-version/node_modules/git-repo-info/index.js:26:32)
    at findRepo (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli-app-version/node_modules/git-repo-version/node_modules/git-repo-info/index.js:46:14)
    at module.exports (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli-app-version/node_modules/git-repo-version/node_modules/git-repo-info/index.js:144:21)
    at version (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli-app-version/node_modules/git-repo-version/index.js:11:14)
    at CoreObject.module.exports.config (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli-app-version/index.js:10:46)
    at CoreObject.superWrapper [as config] (/Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:32:18)
    at /Users/lfender_mbp/source/ghost/core/client/node_modules/ember-cli/lib/models/project.js:226:27
```

This appears related to https://github.com/rwjblue/git-repo-info/issues/27 a dep of ember-cli

### Steps to Reproduce

1. `npm install`
2. `grunt init`

### Technical details:

- Ghost Release 0.11.2
- Node v4.2.6
- Mac Sierra
- GIt 2.10.1 

### To fix

I've gotten around the issue for now by modifying the `package.json` in `core/client` to setup an explicit version of `git-repo-info`.

`"git-repo-info": "1.2.0"`

Then, remove `core/client/node_modules` followed by `grunt init` in project root

**Comment by kevinansfield** [comment]
Hi @lfender6445, yes this is known issue due to https://github.com/rwjblue/git-repo-info/issues/27. It's arisen because git-repo-info doesn't pin it as dependency so npm will pull in the latest broken version despite us being on a pinned version of ember-cli-app-version.

I'm going to close this as the issue is being tracked in the problem repo. Unless you are attempting to hack core I would suggest using the release zips as they contain a pre-built client and shrinkwrapped server dependencies that will avoid these type of development issues.

**Comment by hjdivad** [comment]
@kevinansfield this should be fixed in v1.3.1 of git-repo-info.  Please let me know if this is not the case.
